src/people_page.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+echo "---------------------------------"
+echo "People Page"
+echo "---------------------------------"
+
+page_url="https://${ORGANIZATION_USERNAME}.semaphoreci.com/projects/${SEMAPHORE_PROJECT_ID}/people"
```

Comment:
Do we have SEMAPHORE_PROJECT_NAME ?

Reply:
no, but that can be hardcoded if necessaery, we know that this project is called cont-monitoring

Reply:
@shiroyasha `project_id` is more like our url for easy access projects by id, and `project_name` is more clients url ;)

So I think that we should check clients url.

Reply:
I agree. While writting the impl yesterday, that url was not yet accessible (or maybe the people link was still containing the project id).